include/openssl/ssl.h

```diff
@@ -297,23 +297,27 @@ typedef int (*SSL_verify_cb)(int preverify_ok, X509_STORE_CTX *x509_ctx);
 typedef int (*SSL_async_callback_fn)(SSL *s, void *arg);
 
 /*
- * Some values are reserved until OpenSSL 1.2.0 because they were previously
+ * Some values are reserved until OpenSSL 3.0.0 because they were previously
  * included in SSL_OP_ALL in a 1.1.x release.
- *
- * Reserved value (until OpenSSL 1.2.0)                  0x00000001U
- * Reserved value (until OpenSSL 1.2.0)                  0x00000002U
  */
+
+/* Disable Extended master secret */
+# define SSL_OP_NO_EXTENDED_MASTER_SECRET                0x00000001U
+
+/* Reserved value (until OpenSSL 3.0.0)                  0x00000001U */
+/* Reserved value (until OpenSSL 3.0.0)                  0x00000002U */
```

Comment:
Why not just delete all these reserved values - since they are no longer reserved, i.e. master will become OpenSSL 3.0.0.

Reply:
It seems helpful to have annotations to remind anyone preparing a change for backport of the constraints involved